FinderCompass/src/main/java/atomicstryker/findercompass/common/DefaultConfigFilePrinter.java

```diff
@@ -162,22 +170,27 @@ else if (settingList.isEmpty())
                             }
                             else
                             {
-                                configInts[8] = -1;
+                                configInts[8] = 0;
                             }

```

Comment:
I know it isn't, i had to add it because someone complained about using 0 as wildcard, except he wanted to find only damage 0 blocks